### PR TITLE
CAMEL-18315: camel-hyperledger-aries - Add support for single tenancy…

### PR DESCRIPTION
**Pull request by tdiesler**
… agent

### CODE REVIEW

File: components/camel-hyperledger-aries/src/main/java/org/apache/camel/converter/aries/ConnectionRecordConverter.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/*
```
Comment (davsclaus) [inline components/camel-hyperledger-aries/src/main/java/org/apache/camel/converter/aries/ConnectionRecordConverter.java:1]: There is a duplicated license header here

**Review by davsclaus — APPROVED** [review]

File: components/camel-hyperledger-aries/src/main/java/org/apache/camel/converter/aries/ConnectionRecordConverter.java (hunk 1 above)
Comment (tdiesler) [reply components/camel-hyperledger-aries/src/main/java/org/apache/camel/converter/aries/ConnectionRecordConverter.java:1]: Fixed

**Comment by davsclaus** [comment]
The GH action says there is a test error

**Comment by tdiesler** [comment]
It seems that checkstyle issues (which should be fixed now) are reported as test errors

**Pull request merged**